include/openssl/base.h

```diff
@@ -212,8 +212,6 @@ extern "C" {
 // depending on FIPS mode.
 #define AWSLC_VERSION_NUMBER_STRING "1.0.1"
 
-#if defined(BORINGSSL_SHARED_LIBRARY)
```

Comment:
Why remove? It expands the scope of what was previously encapsulated by the if/def.

Reply:
a failed attempt to address the windows failure